doc-content/enterprise-only/admin-and-config/managing-business-central-customizing-githook-notifications-proc.adoc

```diff
@@ -1,43 +1,48 @@
 [id='managing-business-central-customizing-githook-notifications-proc']
-= Customizing git hook notifications
+= Customizing Git hook notifications
 
-{CENTRAL} provides a mechanism that enables users to receive feedback about git hook executions using customized
-messages based on a hook's exit code.
+{CENTRAL} provides a mechanism that enables users to receive customized Git hook notifications based on the hook exit codes.
 
-.Procedure
+To enable the notification mechanism you must create a `*.properties` file containing the custom messages and then specify the path to that file as the value of the `appformer.git.hooks.bundle` system property.
 
-. To enable notifications, in the `standalone.xml` file set the `appformer.git.hooks.bundle` system property to point to a `*.properties` file containing the custom messages for each exit code.
-+
-For example:
-+
-[source]
-----
-<system-properties>
-  <property name="appformer.git.hooks.bundle" value="/opt/jboss-as/git-hooks-messages/Messages.properties">
-  </property>
-  ...
-</system-properties>
-----
+.Procedure
 
-. To display the messages, each line of the `*.properties` must have the following format:
+. Create the `*.properties` file and add a line for each exit code with a corresponding message in the following format:
 +
 `<exit_code>=<display_message>`
 +
-where, `<exit_code>` is the git hook exit code and `<display_message>` is the custom message that is displayed to a user.
+The `<exit_code>` is the Git hook exit code and the `<display_message>` is the custom message that is displayed to a user.
 +
-Exit code messages with this format appear similar to the following example:
+For example:
 +
 [source]
 ----
-0=Success
-1=Display Message 1
+0=Success! All working as expected.
+1=Warning! Please check the logs and advise your admin.
 .
 .
-31=Display Message 31
+31=Error! Please advise your admin immediately.
 ----
-
++
 [NOTE]
 ====
-Its not necessary to define all the possible exit codes in the *.properties file. Notifications appear only for
-the exit codes defined in the *.properties file.
+It is not necessary to define all the possible exit codes in the *.properties file. Notifications appear only for the exit codes defined in the *.properties file.
+====
++
+[IMPORTANT]
+====
+The notification service only supports the `ISO 8859-1` (`LATIN 1`) character set in the properties file. If you want to use extended characters, please escape them using their Unicode code.
```

Comment:
What does it mean to "escape" them?

Reply:
Unicode character sequences use the escape character.

Reply:
Rephrased to "If you want to use extended characters, please use their escaped Unicode character code sequences."